explorer/site-config/anvil/dev/workspaceTransformer.ts

```diff
@@ -0,0 +1,42 @@
+import * as C from "../../../app/components";
```

Comment:
@BruceRodrigues looks like this is already in explorer/site-config/anvil-catalog/dev/workspaceTransformer.ts